static/src/stylesheets/module/commercial/gustyle/_gu-comcontent.scss

```diff
@@ -35,6 +35,7 @@
     color: #ffffff;
 }
 
+.content--immersive .from-content-api .gu-display__content-title,
```

Comment:
One line in the sand I have in mind is to seldom use the descendant selector. We control the markup, markup & style go hand in hand (markup *is* presentation, especially here), so we should never avoid the opportunity to use more efficient selectors, like the child one.

`.something h2` is a recipe for disaster.